pkg/Microsoft.Private.PackageBaseline/stable.packages.props

```diff
@@ -1342,6 +1342,15 @@
     <StablePackage Include="System.Net.Http.WinHttpHandler">
       <Version>4.0.1</Version>
     </StablePackage>
+    <StablePackage Include="System.Private.DataContractSerialization">
```

Comment:
You shouldn't need to mark the packages as stable as part of this fix. I'm about to go in and mark all the release/1.0.0 packages as stable. The only think that should be needed for your issue is the changes in the baseline.packages.props. Did you rebuild to ensure that correctly puts the right dependency in the package?